.github/workflows/workflow.yml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.8, 3.9, 3.11 ]
+        python-version: [ 3.10, 3.11, 3.12 ]
```

```suggestion
        python-version: [ '3.10', '3.11', '3.12' ]
```
Seems like the GHA is interpreting this as actual numbers, so 3.10 -> 3.1 🤷